e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/ConstraintsTests.java

```diff
@@ -140,6 +140,8 @@ void constraintsSucceedNoViolations() throws IOException {
 
     assertEquals(1, constraintResponses.size());
     assertTrue( constraintResponses.get(0).success());
```

Comment:
Minor, ignorable: spacing
```suggestion
    assertTrue(constraintResponses.get(0).success());
```

Reply:
Formatting should be fixed